sections/02_intro.md

```diff
@@ -34,41 +34,88 @@ learning methods more challenging or less fruitful to apply.
+dimensionality reduction as a strength of neural networks was elucidated, and this has
+motivated designs built to capitalize on this strength [cite autoencoders and word2vec].
+
+Third, several advances in training algorithms have enabled applications of neural networks in ways
+not obviously possible. The number of training strategies for deep learning is growing rapidly and a complete
+reveiw is beyond our scope. But these algorithms can train networks in domains where earlier algorithms struggled. 
+For example, newer optimizers can very efficiently learn using batched training, where only a portion of the data
+needs to be processed at a time. These optimizers more effectively optimize very large weight vectors where many weights are only
+rarely updated. Noise constrastive error has proven particularly useful in 
+modeling language. Reinforcement learning has enabled neural networks to learn how to play games 
+like chess, GO, and poker. Curriculumn learning enables networks to gradually build up expertise to
+solve particularly challenging algorithmic problems. Dropout nodes and layers make networks much more
+robust, even when the number of weights are dramatically increased.
+
+Fourth, the convergence of these factors currently makes deep learning extremely adaptable, and capable 
+of addressing the nuanced differences of each domain to which it is applied. Many of the advances in deep learning
+were first developed for image analysis and text analysis, but the lessons and techniques learnt there
+enable the construction of very powerful models specifically suited to the challenges presented by 
+each unique domain. 
+
+### Deep learning in scientific inquiry
+
+Given the intrinsic flexibility of deep learning, it is important to consider the specific values and goals
+that are particularly important in scientific inquiry.
+
+First, in scientific contexts understanding the patterns in the data may be just as important as fitting the data.
+For this reason, interpretability can be more important here than other domains. Scientific work often 
+aimes to understand the underlying principles behind the data we see, and architectures and techniques that expose the
+non-obvious patterns in the data are particualrly important and very active area of research [cite examples from all sections].
+
+Second, there are important and pressing questions about how to build networks that can efficently represent 
+the underlying logic of the data. This concern of "representability" is important, because it gives insight into
+the structure of scientific data, and when understood can guide the design of more efficent and effective networks. For example,
+one particularly important study was published in Science, which demonstrates that a simple neural network can very efficiently and
+accuratly model (i.e. respresent) a theoretically important quantum mechanical system [http://science.sciencemag.org/content/355/6325/602].
```

Comment:
Glad you introduced this paper, I was looking for a place to refer to it in the review.  I think it is very relevant despite not being a biological example.

Can you please switch to the doi before merging?